tests/MorseCodeTest.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use function PHPUnit\Framework\assertEquals;
+use PHPUnit\Framework\TestCase;
+
+require_once __DIR__ . '/../vendor/autoload.php';
+require_once __DIR__ . '/../ciphers/morseCode.php';
+
+class MorseCodeTest extends TestCase
+{
+    public function testCaesarCipher()
+    {
+        assertEquals('Test 123', decrypt(encrypt('Test 123')));
```

Comment:
Should this be `decode(encode('Test123'))` ?  The CI is failing on encrypt having only 1 argument and it expects 2

```suggestion
        assertEquals('Test 123', decode(encode('Test 123')));
```